package.json

```diff
@@ -21,8 +21,9 @@
   "author": "Christian Chiarulli",
   "license": "ISC",
   "dependencies": {
-    "@vuepress/plugin-search": "^2.0.0-beta.27",
-    "vuepress": "^2.0.0-beta.27",
-    "vuepress-plugin-mermaidjs": "^1.8.1"
+    "@vuepress/client": "^2.0.0-beta.35",
+    "@vuepress/plugin-search": "^2.0.0-beta.35",
+    "vuepress": "^2.0.0-beta.35",
+    "vuepress-plugin-mermaidjs": "^1.9.0"
```

Comment:
Can you move these changes to a separate PR? we can test them and revert them quickly then if there's a problem.

Reply:
Sure thing. Although, lunarvim.org was not usable for me without the new packages added by me. Not sure why.

Reply:
I use `pnpm`, so that might be the source of the issue, but I get this error when running ` lunarvim.org` out of the box.
![Uploading image.png…]()

Reply:
Reverted `package.json` in e6b8170a4ea5b8c4c550d713593077502291c1bc

Reply:
To clarify, I don't have a problem with updating package.json, in fact, we should probably do that anyway for security reasons. I just thought it might be better to do it separately.

Reply:
@kylo252 I understand that. Opening up a new PR for the bump and addition in package.json :)

Reply:
Turns out vuepress 2.0 requires some config of pnpm